### PR TITLE
Model/implicit model

### PR DESCRIPTION
**Pull request by sondrenaerland**
The code runs, tested on rproblem2 and rproblem8. Runs extremely slow, as expected. Takes about 1 hour before the model reaches the final stage of the optimization (Gurobi log-file starts document lines of iterations).

It might be relevant to look for shortcuts to remove unnecessary data/variables/parts of sets, as we did last year. I do however think that we only are to run tests using the implicit model on very simplified test cases. Therefore, the code could function as it is now.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -152,7 +153,10 @@ def get_candidate_solution(self):
     def run_esp(self):
         """ Runs ESP, with an optional presolve with SDP """
 
-        logger.info(f"Running ESP in mode {self.mode} with {len(self.esp.shifts_set['shifts'])}")
+        if self.mode != "implicit":
```
Comment (ekallevik) [inline main.py:156]: Should be 
```python
if self.mode != "implicit" and self.mode != 3
```

File: main.py [hunk 2]
```diff
@@ -61,7 +62,7 @@ def __init__(self, problem="rproblem3", mode="feasibility", with_sdp=True, log_n
         self.log_to_console = 1
 
         self.sdp = None
-        if with_sdp:
+        if with_sdp and self.mode != "implicit":
```
Comment (ekallevik) [inline main.py:65]: Should be 
```python
if with_sdp and (self.mode != "implicit" and self.mode != 3)
```

File: utils/const.py [hunk 3]
```diff
@@ -7,6 +7,9 @@
 DEFAULT_DAILY_REST_HOURS = 8
 DEFAULT_WEEKLY_REST_HOURS = 36
 DEFAULT_DAILY_OFFSET = 4
+DEFAULT_WEEKLY_OFFSET = 0
+
+LIMIT_CONSECUTIVE_DAYS = 5
```
Comment (ekallevik) [inline utils/const.py:12]: Awesome! 👏

File: main.py (hunk 1 above)
Comment (sondrenaerland) [reply main.py:156]: or i stedet for and, eller?

Comment (ekallevik) [reply main.py]: Nope, this is the correct way. You want this code to run if you have chosen  mode that is not _implicit_ / 3.

|  Input/Result | AND | OR |
| ------------- | ------------- | ------------- |
| _implicit_  | `False` due to first term | `True` due to second term  |
| 3 | `False` due to second term  | `True` due to first term |
| Anything else  | `True`  | `True`  |

**Pull request merged**